lbc.tex

```diff
@@ -118,15 +118,15 @@ \section{Specified Lateral Boundary Conditions}
  \centering
   \includegraphics *[width=4.5in]{figures/lbc_zones.pdf}
   \caption{\label{figure:spec} Specified and relaxation zones for a grid
-with a single specified row and column, and four rows and columns for the
+with a single-specified row and column, plus four rows and columns for the
```

Comment:
@davegill  Should we say three here? Or should we add the most inner row and column use model data only?

Reply:
@weiwangncar 
Now this is more clearly stated.